main/res/values/strings.xml

```diff
@@ -299,7 +299,7 @@
     <string name="info_select_logimage_cancelled">Image selection or capture was cancelled.</string>
     <string name="info_cache_saved">The cache has been stored locally</string>
     <string name="warn_calculator_state_save">Be sure to save waypoint if you want to keep the calculator state.</string>
-    <string name="feature_info_offline_counter">You have some caches with an offline saved log, which might not yet be logged online. For your comfort the count of these logs is displayed as \"+ x\" behind your number of found caches in the footer at c:geo\'s main page. \n\nMoreover these \"x\" logs will be added to the \"NUMBER\" template you might use in your log signature. \n\nIf you don\'t like this counter extension, you can disable it under \"Settings\" > \"Logging\" and/or use the \"NUMBER (without offline logs)\" template instead. \n\nTo easily find remaining offline logs, go to the list \"History\" and filter for \"With personal data\" > \"With offline log\".</string>
+    <string name="feature_info_offline_counter">You have some caches with an offline saved log, which might not yet be logged online. For your convenience the count of these logs is displayed as \"+ x\" behind your number of found caches in the footer at c:geo\'s main page. \n\nMoreover these \"x\" logs will be added to the \"NUMBER\" template you might use in your log signature. \n\nIf you don\'t like this counter extension, you can disable it under \"Settings\" > \"Logging\" and/or use the \"NUMBER (without offline logs)\" template instead. \n\nTo easily find remaining offline logs, go to the list \"History\" and filter for \"With personal data\" > \"With offline log\".</string>
```

Comment:
With the bottom navigation and a redesigned dashboard, the dialog will probably be removed anyway, as the new style will be way more self-explaining. So I don't know if it makes sense to change anything here...?

![signal-2021-07-21-195912](https://user-images.githubusercontent.com/64581222/127562801-1bd917df-4ec1-4242-b740-69c3a0dca4fc.png)